CONTRIBUTING.md

```diff
@@ -1,60 +1,123 @@
 
-The ‘untriaged’ label is assigned automatically. During the triage process, the documentation team will add the appropriate labels, assign the issue to a technical writer, and prioritize the request. We may follow up with you for additional information. 
+## Contribute content
 
+There are two ways to contribute content:
 
-**Create a pull request**
+- [Editing files directly in GitHub](#editing-files-in-github): Best for small changes like fixing a typo or adding a parameter. This approach does not require cloning the repository and does not let you test the documentation.
+- [Making changes locally and pushing to GitHub](#making-changes-locally): Best for changes you want to test locally first, like adding a table or section, or reorganizing pages. This approach requires setting up a local version of the repository and lets you test documentation.
 
-If you’d like to make a change directly to the content, create a pull request. If it’s a quick fix, we should be able to release the update quickly. Bigger requests might take a bit of time for us to review. 
+The workflow for contributing documentation is no different than the one for contributing code:
 
-Note that a pull request requires DCO sign off before we can merge it. You can use the -s command line option to append this automatically to your commit message, for example $ git commit -s -m 'This is my commit message'. For more information, see https://github.com/apps/dco.
+- Make your changes
+- Build locally to check your work (only possible if you are making changes locally)
+- Submit a [pull request](https://github.com/opensearch-project/documentation-website/pulls) (PR)
+- Maintainers review and merge your PR
 
-Before submitting, make sure to:
+If the change requires significant work, open an issue where we can first discuss your request.
 
-* Work against the latest source on the main branch.
-* Check existing open and recently merged pull requests to ensure that someone else hasn't addressed the problem already.
-* If the change requires significant work, open an issue where we can first discuss your request.
+### Before you start
 
-To create a pull request:
+Before contributing content, make sure to read the following resources:
+- [README](README)
+- [OpenSearch Project Style Guidelines](STYLE_GUIDE)
+- [API Style Guide](API_STYLE_GUIDE)
+- [Formatting Guide](FORMATTING_GUIDE) 
 
-1. Fork the repository.
-2. Modify the source. Make sure to focus on the specific change you are contributing. For example, if you also reformat the code, it will be hard for us to focus on your change.
-3. Test the code. Make sure that any local tests pass.
-4. Commit to your fork using clear commit messages.
-5. Create the [pull request](https://github.com/opensearch-project/documentation-website/pulls) and save it, making sure to answer the default questions in the pull request template.
+### Editing files in GitHub
```

Comment:
I'm confused about these headings. I'll reach out offline.

Reply:
@hdhalter I made some changes to the structure of the doc. Please review again. Thanks.